Resources/Prototypes/_Scp/Entities/Markers/Spawners/Random/posters.yml

```
@@ -46,33 +47,130 @@
+    - PosterLegitBotanyFood
+    - PosterLegitHotDonkExplosion
+    - PosterLegitEatMeat
+    - PosterLegitMoreBread
+    - PosterLegitPieSlice
+    - PosterLegitPizzaHope
+    - PosterLegitMedicate
+    - PosterLegitNoTouching
+    - PosterLegitChknDnnr
+    - PosterContrabandSaucerNumberOne
+    - PosterContrabandBreadLies
+    - PosterLegitCleanliness # poster3_legit
+    - PosterLegitHelpOthers # poster4_legit
+    - PosterLegitScience # poster7_legit
+    - PosterLegitWalk # poster10_legit
+    - PosterLegitUeNo # poster14_legit
+    - PosterLegitGetYourLEGS # poster15_legit
+    - PosterLegitDoNotQuestion # poster16_legit
+    - PosterLegitWorkForAFuture # poster17_legit
+    - PosterLegitSoftCapPopArt # poster18_legit
+    - PosterLegitSafetyInternals # poster19_legit
+    - PosterLegitSafetyEyeProtection # poster20_legit
+    - PosterLegitSafetyReport # poster21_legit
+    - PosterLegitReportCrimes # poster22_legit
+    - PosterLegitFoamForceAd # poster24_legit
+    - PosterLegitCohibaRobustoAd # poster25_legit
+    - PosterLegitFruitBowl # poster27_legit
+    - PosterLegitNanomichiAd # poster30_legit
+    - PosterLegitHighClassMartini # poster32_legit
+    - PosterLegitNoERP # poster34_legit
+    - PosterLegitDickGumshue # poster36_legit
+    - PosterLegitThereIsNoGasGiant # poster37_legit
+    - PosterLegitAnatomyPoster # poster40_legit
+    - PosterLegitMime # poster41_legit
+    - PosterLegitVacation # poster48_legit
+    - PosterLegitPeriodicTable # poster49_legit
+    - PosterContrabandAriralInSpace
+    - PosterContrabandSmoke # poster7_contraband
+    - PosterContrabandMissingGloves # poster9_contraband
+    - PosterContrabandEAT # poster14_contraband
+    - PosterContrabandTools # poster15_contraband
+    - PosterContrabandRedRum # poster30_contraband
+    - PosterContrabandBeachStarYamamoto # poster52_contraband
+    - PosterContrabandMissingSpacepen # poster63_contraband
+    # SCP
+    - PosterScp # scp
+    - PosterBreath # breath
+    - PosterScpYours # nt
+    - PosterLook # scp173
+    - Poster217 # scp217
+    - PosterSaulAgency # saul
+    - PosterChanges # changes
+    - PosterLooseLipsLoseScips # loose_lips_lose_scips
+    - PosterScpHey # poster1_legit
+    - PosterScpBonFire # poster2_legit
+    - PosterScpShodan # poster3_legit
+    - PosterScpVal # poster4_legit
+    - PosterScpBand # poster5_legit
+    - PosterScpLandscape # poster6_legit
+    - PosterScpSinger # poster7_legit
+    - PosterScpColleague # poster8_legit
+    - PosterScpGirl # poster9_legit
+    - PosterScpScientist # poster10_legit
+    - PosterScpScientist2 # poster11_legit
+    - PosterScpIHaveNoMouthAndIMustScream # poster12_legit
+    - PosterScpDoctor # poster13_legit
+    - PosterScpNun # poster14_legit
+    - PosterScpGirl2 # poster15_legit
+    - PosterScpGirl3 # poster16_legit
+    - PosterScpGirl4 # poster17_legit
+    - PosterScpEngineerGaming # poster18_legit
+    - PosterScpSecurity # poster19_legit
+    chance: 0.9
```

```suggestion
    chance: 0.5
```

Аналогично с нелегальными плакатами.

---

Аналогично выше